test/srt/test_serving_throughput.py

```diff
@@ -55,21 +56,30 @@ def run_test(self, disable_radix_cache, disable_flashinfer, chunked_prefill_size
             kill_child_process(process.pid)
 
         assert res["completed"] == num_prompts
+        return res
 
     def test_default(self):
-        self.run_test(
+        res = self.run_test(
             disable_radix_cache=False,
             disable_flashinfer=False,
             chunked_prefill_size=-1,
         )
 
+        if os.getenv("SGLANG_IS_IN_CI", "false") == "true":
+            # A100 performance
+            assert res["output_token_throughput"] >= 1300
```

Comment:
Why does it show a KeyError?

ref https://github.com/sgl-project/sglang/actions/runs/10348574527/job/28641104727

```
100%|██████████| 400/400 [04:40<00:00,  1.42it/s]
/usr/lib/python3.10/subprocess.py:1072: ResourceWarning: subprocess 3619578 is still running
  _warn("subprocess %s is still running" % self.pid,
ResourceWarning: Enable tracemalloc to get the object allocation traceback
E
======================================================================
ERROR: test_default (test_serving_throughput.TestServingThroughput)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/data/zhyncs/runner-v1/actions-runner/_work/sglang/sglang/test/srt/test_serving_throughput.py", line 70, in test_default
    assert res["output_token_throughput"] >= 1300
KeyError: 'output_token_throughput'
```

Reply:
fixed.